boards/raspberrypi/rpi_pico2/rpi_pico2-common.dtsi

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) 2024 Andrew Featherstone
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <freq.h>
+
+#include <raspberrypi/rpi_pico/rp2350a.dtsi>
+#include "rpi_pico2-pinctrl.dtsi"
+#include <zephyr/dt-bindings/pwm/pwm.h>
+
+#include <zephyr/dt-bindings/i2c/i2c.h>
+
+
+/ {
```

Comment:
All of this section is common to all Pico and Pico 2 boards. It could/should be made into something common to emphasise this.